### ISSUE TITLE
Invalid and duplicate bugs should not be counted as bugs in the statistics

### ISSUE THREAD

**Issue by gsmet** [rsvoboda/gh-exporter#5]
Invalid are not bugs.
Duplicate are already counted once, no need to count them twice.

**Comment by rsvoboda** [comment]
This can be achieved with current code if `gh.label.bug` property is set to `kind/bug+-label:triage/invalid+-label:triage/duplicate` 
See https://github.com/rsvoboda/gh-exporter/blob/master/src/main/java/io/quarkus/qe/metrics/GHRepositoryAdvancedMetrics.java#L39

The same should be probably applied to enhancements and extension proposals :/ 

This classification of "real" bugs is really valid for closed issues, usually when duplicate or invalid issue is spotted it gets closed. And on the other hand people usually look at open issues. 

So I don't really lean towards changing the current defaults, I'm thinking of adding metrics for invalid and duplicate (closed) issues. I think it can be quite interesting to see trends for invalid and duplicate issues.

Things like `label:kind/bug+-label:triage/invalid+-label:triage/duplicate` I would handle via https://github.com/rsvoboda/gh-exporter/issues/4. It's drill down to see the "real" bugs, but the messy items around can be interesting too.

To sum up, I would keep (and enhance a bit) current simple/naive view on GH issues ++  introduce more focused view and metrics via #4

**Comment by rsvoboda** [comment]
Example of deploy - https://github.com/rsvoboda/gh-jira-metrics/commit/f2a9d6a6f0ff0961a971440b824bd14a8187335c
